src/keria/app/aiding.py

```diff
@@ -155,9 +155,6 @@ def on_put(self, req, rep, caid):
         ctrlHab = agent.hby.habByName(caid, ns="agent")
         ctrlHab.rotate(serder=rot, sigers=[coring.Siger(qb64=sig) for sig in sigs])
 
-        if not self.authn.verify(req):
```

Comment:
This should not have been removed.

Reply:
The deletion of those two lines had no sense in that PR. I added again. I probably removed while adding and trying  ne tests.